sci-misc/netlogo-bin/netlogo-bin-6.2.0.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop java-pkg-2 xdg
+
+MY_PN="NetLogo"
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Cross-platform multi-agent programmable modelling environment"
+HOMEPAGE="https://ccl.northwestern.edu/netlogo/"
+SRC_URI="
+	https://netlogoweb.org/assets/images/desktopicon.png -> ${PN//-bin}.png
+	amd64? ( https://ccl.northwestern.edu/netlogo/${PV}/${MY_P}-64.tgz )
+	x86? ( https://ccl.northwestern.edu/netlogo/${PV}/${MY_P}-32.tgz )
+"
+S="${WORKDIR}/${MY_PN} ${PV}"
+
+LICENSE="netlogo GPL-2 LGPL-2.1 LGPL-3 BSD Apache-2.0"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+RDEPEND="
+	>=virtual/jre-1.8
```

Comment:
Why limit to 1.8? Doesn't it work with [>=virtual/jre-1.8:*](https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE) ?

Reply:
It should work with anything newer then 1.8, but AFAIK `>=virtual/jre-1.8` doesn't restrict it to 1.8 anyway right?